tests/bud.bats

```diff
@@ -1641,3 +1641,12 @@ load helpers
   touch "$DIR"/Dockerfile
   run_buildah 0 bud "$DIR"/Dockerfile
 }
+
+@test "bud with additional device" {
+  target=alpinepriv
+  run_buildah bud --signature-policy ${TESTSDIR}/policy.json --device /dev/fuse -t ${target} -f ${TESTSDIR}/bud/device/Dockerfile ${TESTSDIR}/bud/device
```

Comment:
To avoid potential regressions, can we also try that with different modes (rwm)?